### DISCUSSION TITLE
Run cruise control in a light way

### DISCUSSION THREAD

**Discussion by bingkunyang** [linkedin/cruise-control#2227] [Q&A]
Hi, I am here just want to ask if there’s a way to trigger/run the cruise control in a light way? I’m asking because sometimes our load increase and some brokers’ disks are growing quickly. And because our partition is so big and it takes a lot of time to even move one. So running the balance with cruise control would take a lot of time. So if there a way that we can run the cruise control just to move the partition from the broker whose disk are above certain threshold (like above 95%) to the one that has lower utilization? In this way we can quickly mitigate the issue, without running the whole balance for cruise control? Your reply would be much appreciated

**Comment by kyguy** [comment]
Cruise Control is designed to automate partition rebalancing across the cluster based on a variety of factors. However, if you've identified a single problematic partition and only need to move that one, using Cruise Control might be unnecessary overhead. In such cases, manually reassigning the partition with the kafka-reassign-partitions.sh tool is likely a more straightforward solution.
